src/routes/components/components.json

```diff
@@ -2705,5 +2705,151 @@
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "animations", "typescript"]
+	},
+	{
+		"title": "svelte-headlessui",
+		"url": "https://captaincodeman.github.io/svelte-headlessui/",
+		"repository": "https://github.com/captaincodeman/svelte-headlessui",
+		"description": "A very lightweight unofficial implementation of Tailwind HeadlessUI",
+		"npm": "svelte-headlessui",
+		"addedOn": "2023-08-16",
+		"category": "Design System",
+		"tags": ["components and libraries", "component sets", "typescript"]
+	},
+	{
+		"title": "svelte-color-select",
+		"url": "https://captaincodeman.github.io/svelte-color-select/",
+		"repository": "https://github.com/captaincodeman/svelte-color-select",
+		"description": "OKHSV Color Selection component for Svelte using OKLab perceptual colorspace",
+		"npm": "svelte-color-select",
+		"addedOn": "2023-08-16",
+		"category": "",
+		"tags": ["components and libraries", "typescript"]
+	},
+	{
+		"title": "svelte-form-helper",
+		"url": "https://captaincodeman.github.io/svelte-form-helper/",
+		"repository": "https://github.com/captaincodeman/svelte-form-helper",
+		"description": "Lightweight helper for form validation with Svelte",
+		"npm": "svelte-form-helper",
+		"addedOn": "2023-08-16",
+		"category": "Forms & User Input",
+		"tags": [
+			"inputs and widgets",
+			"forms and validation",
+			"validation",
+			"typescript",
+			"components and libraries"
+		]
+	},
+	{
+		"title": "svelte-theme-select",
+		"url": "https://captaincodeman.github.io/svelte-theme-select/",
+		"repository": "https://github.com/captaincodeman/svelte-theme-select",
+		"description": "Customizable Svelte components for theme selection (light mode / dark mode) inspired by TailwindCSS",
+		"npm": "svelte-theme-select",
+		"addedOn": "2023-08-16",
+		"category": "",
+		"tags": ["components and libraries", "typescript"]
+	},
+	{
+		"title": "svelte-pan-zoom",
+		"url": "https://captaincodeman.github.io/svelte-pan-zoom/",
+		"repository": "https://github.com/captaincodeman/svelte-pan-zoom",
+		"description": "HTML Canvas Element Pan and Zoom action for Svelte",
+		"npm": "svelte-pan-zoom",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "images", "typescript"]
+	},
+	{
+		"title": "svelte-deep-zoom",
+		"url": "https://captaincodeman.github.io/svelte-deep-zoom",
+		"repository": "https://github.com/captaincodeman/svelte-deep-zoom",
+		"description": "Svelte component to render interactive Deep Zoom images (tiled image pyramids). Similar to Open SeaDragon but smaller",
+		"npm": "svelte-deep-zoom",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "images", "typescript"]
+	},
+	{
+		"title": "svelte-resize-observer-action",
+		"url": "https://captaincodeman.github.io/svelte-resize-observer-action/",
+		"repository": "https://github.com/captaincodeman/svelte-resize-observer-action",
```

Comment:
this one has less than 15 stars, so perhaps we should hold off on it https://github.com/svelte-society/sveltesociety.dev/pull/442